agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -2147,7 +2157,17 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
 
         send_cmdu_to_controller(cmdu_tx);
 
-        break;
+        beerocks_message::sClientMonitoringParams monitoring_params;
+        monitoring_params.mac = notification_in->params().mac;
+        monitoring_params.bridge_4addr_mac =
+            network_utils::mac_from_string(backhaul_params.bridge_mac);
+        monitoring_params.ipv4    = network_utils::ipv4_from_string(network_utils::ZERO_IP_STRING);
+        monitoring_params.channel = backhaul_params.backhaul_channel; // TODO: unused?
+        monitoring_params.vap_id  = notification_in->params().vap_id;
+        monitoring_params.is_ire  = 0;
```

Comment:
This looks wrong. I don't understand the logic completely (perhaps @morantr or @vitalybu can shed some light), but IRE stands for "Intelligent range extender", which means its a repeater (well, range extender differs from repeater in that it has 3 radios, meaning a dedicated radio for backhaul).

From the looks of it, it looks like the whole IRE VS messaging path here should be removed. It is sending a notification to the ap manager that the connected client is a 4addr client, and this whole path is now dead code as it seems. But this should be removed in a separate PR.

For now, I would just remove the is_ire from here completely (in a separate commit).

Reply:
To be honest, I don't have much confidence in these values.
According to the logs from `./test_flows.py`, `ipv4` and `bridge_4addr_mac` are always zero.
Setting `is_ire` to non-zero value breaks `client_steering_dummy` test.

Reply:
That's exactly what Tomer is saying: the is_ire != 0 part should be removed entirely.

Reply:
@tomereli , @arnout  Shall I remove the `ACTION_APMANAGER_CLIENT_IRE_CONNECTED_NOTIFICATION` , the list of IREs (`inserted_ire`) managed by ap manager and all the code that uses the list?

Or just remove `is_ire != 0` path from `ACTION_APMANAGER_CLIENT_ASSOCIATED_NOTIFICATION`?